substrate-archive-backend/src/runtime_version_cache.rs

```diff
@@ -120,12 +120,12 @@ impl<B: BlockT, D: ReadOnlyDB + 'static> RuntimeVersionCache<B, D> {
 	where
 		NumberFor<B>: Into<u32>,
 	{
-		let versions = self.find_versions(blocks.as_slice())?;
+		let versions = self.find_versions(&blocks)?;
```

Comment:
dq: are these exactly equivalent?

Reply:
They are, yes. It's because of how `AsRef` is defined for `Vec<T>` and `[T]` (I think; others can correct me).